src/uwtools/drivers/chgres_cube.py

```diff
@@ -0,0 +1,133 @@
+"""
+A driver for chgres_cube.
+"""
+
+import os
+import stat
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict
+
+from iotaa import asset, dryrun, task, tasks
+
+from uwtools.config.formats.nml import NMLConfig
+from uwtools.drivers.driver import Driver
+from uwtools.strings import STR
+from uwtools.utils.tasks import file
+
+
+class ChgresCube(Driver):
+    """
+    A driver for chgres_cube.
+    """
+
+    _driver_name = STR.chgrescube
+
+    def __init__(
+        self, config_file: Path, cycle: datetime, dry_run: bool = False, batch: bool = False
+    ):
+        """
+        The driver.
+
+        :param config_file: Path to config file.
+        :param cycle: The forecast cycle.
+        :param dry_run: Run in dry-run mode?
+        :param batch: Run component via the batch system?
+        """
+        super().__init__(config_file=config_file, dry_run=dry_run, batch=batch)
+        self._config.dereference(context={"cycle": cycle})
+        if self._dry_run:
+            dryrun()
+        self._cycle = cycle
+
+    # Workflow tasks
+
+    @task
+    def namelist_file(self):
+        """
+        The namelist file.
+        """
+        fn = "fort.41"
+        yield self._taskname(f"namelist file {fn}")
+        path = self._rundir / fn
+        yield asset(path, path.is_file)
+        vals = self._driver_config["namelist"]["update_values"]["config"]
+        inputs = [
+            ("data_dir_input_grid", "atm_files_input_grid"),
+            ("data_dir_input_grid", "grib2_file_input_grid"),
+            ("data_dir_input_grid", "sfc_files_input_grid"),
+            "mosaic_file_target_grid",
+            "varmap_file",
+            "vcoord_file_target_grid",
+        ]
+        input_paths = []
+        for item in inputs:
+            if isinstance(item, str):
+                input_paths += [Path(vals[item])]
+            else:
+                input_paths += [Path(vals[item[0]]) / vals[item[1]]]
```

Comment:
I am giving `config_files` a try here since we're really looking for just those config settings. Otherwise, I think `update_vals` might be okay. Thoughts?

Reply:
I like `config_files`.